plugins/Makefile

```diff
@@ -35,16 +41,16 @@ godeps:
 	$(GO) get github.com/hyperledger/fabric-chaincode-go/shimtest
 
 .PHONY: ecc-vscc
-ecc-vscc: $(ECC_VSCC_SRC)
-	$(GO) build -o $(ECC_VSCC_OUT) -buildmode=plugin $^
+ecc-vscc: $(ECC_VSCC_SRC) $(ATTESTATION_PROVIDER)
+	$(GO) build -o $(ECC_VSCC_OUT) -buildmode=plugin $(ECC_VSCC_SRC)
```

Comment:
Why making attestation provider a dependency here? Looking into `ecc_validation_logic.go` there is no import for `ercc/attestation`?!

Reply:
Can't remember what failed but something definitely failed ...

Reply:
I guess it failed for `ercc-vscc`.

Reply:
ok, seems to work fine when having dependency only on ercc-vscc.